__tests__/integration/helpers/wallet-precalculation.helper.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { promises as fs } from 'fs';
+import { Address, Script } from 'bitcore-lib';
+import wallet from '../../../src/wallet';
+import walletUtils from '../../../src/utils/wallet';
+import { NETWORK_NAME } from '../configuration/test-constants';
+
+/**
+ * @typedef PrecalculatedWalletData
+ * @property {boolean} isUsed Indicates if this wallet was already used
+ * @property {string} words 24-word seed
+ * @property {string[]} addresses List of pre-calculated addresses
+ */
+
+export const precalculationHelpers = {
+  /**
+   * @type WalletPrecalculationHelper
+   */
+  test: null
+};
+
+export const multisigWalletsData = {
```

Comment:
The reason this is kept outside this json file is similar to the described above for the genesis wallet: we currently treat this `multisigWalletsData` object as the _only_ multisig wallet pre-calculated for our tests.

And since the structure of the data we need for testing it is different from a "common" wallet, I believe we would have to refactor the `precalculated-wallets.json` to accomodate it: having an array of "common" wallets and an array of "multisig" wallets, for example.

For the needs we have now, it doesn't seem a good investment of time to me. But I agree that it will be a relevant refactor as soon as we increase the test complexity and try to test multiple multisig wallets, for example.

Do you agree?